README.md

```diff
@@ -4,24 +4,11 @@
 
 # Specifications for bioimage.io
 
-This repository contains specifications defined by the bioimage.io community. These specifications are used for defining fields in YAML 1.2 files which should be named `bioimageio.yaml`. Such a bioimageio.yaml --- along with files referenced in it --- can be downloaded from or uploaded to the [bioimage.io website](https://bioimage.io) and may be produced or consumed by bioimage.io-compatible consumers (e.g. image analysis software like ilastik).
+This repository contains the specifications of the standard format defined by the bioimage.io community for the content (i.e., models, datasets and applications) in the [bioimage.io website](https://bioimage.io). Each item in the content is always described using a YAML 1.2 file named `bioimageio.yaml`. Such a `bioimageio.yaml` --- along with the files referenced in it --- can be downloaded from or uploaded to the [bioimage.io website](https://bioimage.io) and may be produced or consumed by bioimage.io-compatible consumers (e.g., ilastik). The following specifications define each of the fields in the `bioimageio.yaml`. 
 
-bioimage.io-compatible resources must fulfill the following rules:
+## Versioned specifications
 
-Note that the Python package PyYAML does not support YAML 1.2 .
-We therefore use and recommend [ruamel.yaml](https://ruamelyaml.readthedocs.io/en/latest/).
-For differences see <https://ruamelyaml.readthedocs.io/en/latest/pyyaml>.
```

Comment:
could we keep this information on the difference of PyYAML and ruamel.yaml?

Reply:
ufa, I thought I added it to the recommendations. Let me change it